File: python/sparklabs/batch/sales/rdd/spark_salting.py
import random

from sparklabs.batch.sales.rdd import utils


def run() -> None:
    for row in utils.sc.parallelize(range(20)).map(
        lambda identifier: (identifier, int(random.random() * 10))
    ).collect():
        print(row)


if __name__ == "__main__":
    run()

File: scala/src/main/scala/io/devorbit/sparklabs/batch/sales/rdd/SparkSalting.scala
package io.devorbit.sparklabs.batch.sales.rdd

object SparkSalting extends App {
  RddUtils.orders.context.parallelize(0 until 20)
    .map(id => (id, (scala.util.Random.nextDouble() * 10).toInt))
    .collect()
    .foreach(println)
}
